### ISSUE TITLE
Bullets with a checkbox and wrapping text show a bullet on the last wrapping line

### ISSUE THREAD

**Issue by psnider** [atom/markdown-preview#142]
This markdown:

```
- [ ] aaa bbbb ccccc ddddd eeeee fff gggggggg hhhhhhh iiiiii,jjjj uuuu wwww xxxx zzzz 
```

renders as:

![screen shot 2014-10-19 at 9 26 01 am](https://cloud.githubusercontent.com/assets/940931/4693810/a546b280-57ad-11e4-84a8-87e580c3f50b.png)

The bullet will stay on the last wrapping fragment as you change the width of your window.

**Comment by nitrocode** [comment]
I see bullets before each checkbox generated by `- [ ]` and by `- [x]`

**Comment by harrisitservices** [comment]
This is definitely an issue. - [ ] is showing bullets in front of the checkbox. I looked at the html css and list does not have the proper class as it does on github. On github if you do - [ ] the ul class is task-list-item and the li class is task-list-item-checkbox. In markdown preview the ul and li are both without class which results in the bullet showing in front of the checkbox. Can this be fixed soon please?

**Comment by intcreator** [comment]
This is still an issue.  I tried other ways of writing checkboxes but none of them rendered besides the syntax used above.  However, that syntax still renders bullet points when there should be none.

**Comment by 50Wliu** [comment]
This was fixed by #428 and will be available starting with Atom 1.12.0.

**Comment by MCDELTAT** [comment]
I'm still having this issue in 1.18.0 on Linux. I'll have a more thorough look at it later today.

**Comment by vineeshvs** [comment]
> This was fixed by #428 and will be available starting with Atom 1.12.0.

Hi,
I tired the following in Bitbucket README.md

- [ ] Push my commits to GitHub
- [ ] Open a pull request

Still getting bullets instead of checkboxes. Anyway to apply the fixes in #468 in this case from my side?

**Comment by 50Wliu** [comment]
@vineeshvs could you please open a new issue?
